org.eclipse.jdt.ls.core/gradle/android/init.gradle

```diff
@@ -59,20 +58,27 @@ class JavaLanguageServerAndroidPlugin implements Plugin<Project> {
                 }
                 eclipseModel.synchronizationTasks(synchronizationTasks)
             }
+            List<Object> androidSourceSets = getAndroidSourceSets(project)
             // Get project configurations from supported sourceSets names and configuration keys,
             // and add them to eclipseModel.classpath.plusConfigurations
-            addPlusConfiguration(project, eclipseModel)
+            addPlusConfiguration(project, eclipseModel, androidSourceSets)
             eclipseModel.classpath.setDownloadSources(true)
             // Remove JDK container since android project should use embedded JDK types included in android.jar
-            eclipseModel.classpath.containers.removeIf(container -> container.contains("JavaSE"))
+            List<String> containers = new ArrayList<>()
+            for (String containerName : eclipseModel.classpath.containers) {
+                if (containerName.contains("JavaSE")) {
+                    containers.add(containerName)
+                }
+            }
+            eclipseModel.classpath.containers.removeAll(containers)
```

Comment:
Is there any reason that change the previous `removeIf` to this `removeAll` version?